.eslintrc.json

```diff
@@ -1,19 +1,28 @@
 {
   "extends": "airbnb-base",
+  "env": {
+    "es6": false,
+    "browser": true,
+    "jquery": true
+  },
   "rules": {
     "camelcase": 0,
     "comma-dangle": 0,
     "dot-notation": 0,
     "func-names": 0,
-    "no-console": 0,
-    "no-extra-semi": 0,
+    "no-console": ["warn", {
+        "allow": ["warn"]
+    }],
+    "no-extra-semi": "warn",
```

Comment:
Why a warning instead of an error for this rule? Do we expect to use extra semicolons while developing?

Reply:
Given the discussion in #2847, I think the general consensus is to keep semicolons and only remove the ones that are completely unnecessary.

As for the warning instead of an error, I think that was an arbitrary decision :sweat_smile: I feel like the warnings should be used for stuff like this, whilst errors should be used when variable shadowing, unused vars or no `var` usage issues are present, but I'm completely willing to discuss what's the better choice for this. Let me know what you think! :raised_hands:

Reply:
@aitbw but in #2847 there isn't a discussion about extra semicolons :confused: for what I read `no-extra-semi` rule is for code like this: `;;` while the semicolons toggle rule is named `semi`, isn't it?

Reply:
https://eslint.org/docs/rules/no-extra-semi 

I think I'm reading the same way as @mdesantis. This is a non-blocker for me, but I would go ahead and make it an error personally.

Reply:
You're right, @mdesantis. Fixing it as we speak! :wink: